windows/internal/rnn_smoke.py

```diff
@@ -0,0 +1,13 @@
+r"""
+It's used to check basic rnn features in cuda.
```

Comment:
Same here for this line and the following line

Reply:
thanks. updated